lib/transaction.ex

```diff
@@ -31,6 +31,67 @@ defmodule Elixium.Transaction do
     %{outputs: outputs}
   end
 
+  @doc """
+  Creates a correct tx id
+  """
+  @spec create_tx_id(List, String.t()) :: String.t()
+  def create_tx_id(tx, tx_timestamp) do
+    calculate_hash(tx) <> tx_timestamp
+    |> Utilities.sha_base16()
+  end
+
+  @doc """
+  Creates a singature list based off the input being passed in and the corresponding private key
+  """
+  @spec create_sig_list(Map, Map) :: Tuple
+  def create_sig_list(input, transaction) do
```

Comment:
This should only take in a list of addresses, rather than a whole input. Also, this shouldn't be called 'create_sig_list' because it only returns a single tuple and not a list

Reply:
Fixed